bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/preferences/BonitaUserXpPreferencePage.java

```diff
@@ -157,7 +157,7 @@ public void run(final IProgressMonitor monitor) throws InvocationTargetException
     public void propertyChange(final PropertyChangeEvent event) {
         if (event.getSource().equals(port)) {
             if (port.isValid()) {
-                newPort = Integer.valueOf((String) event.getNewValue());
+                newPort = port.getIntValue();
```

Comment:
we don't use anymore the "newValue" sent by the event? while reading it seems to me that we are reusing the old port value